packages/devtools_app/lib/src/screens/network/network_controller.dart

```diff
@@ -114,22 +114,23 @@ class NetworkController extends DisposableController
   /// timeline events.
   late int _timelineMicrosOffset;
 
-  /// The last timestamp at which HTTP and Socket information was refreshed.
-  int lastRefreshMicros = 0;
+  /// The last time at which HTTP information was refreshed.
+  DateTime lastHttpDataRefreshTime = DateTime.fromMicrosecondsSinceEpoch(0);
+
+  /// The last timestamp at which Socket information was refreshed. This
```

Comment:
nit: new line after first sentence of dartdoc

Reply:
Done.